modules/passkey/contracts/test/TestWebAuthnSingletonSigner.sol

```diff
@@ -2,52 +2,139 @@
 pragma solidity >=0.8.0;
 
 import {SignatureValidator} from "../base/SignatureValidator.sol";
+import {ISafe} from "../interfaces/ISafe.sol";
 import {P256, WebAuthn} from "../libraries/WebAuthn.sol";
 
 /**
  * @title WebAuthn Singleton Signer
- * @dev A contract for verifying WebAuthn signatures for multiple accounts.
+ * @dev A contract for verifying WebAuthn signatures for multiple accounts. This contract uses
```

Comment:
Incomplete sentence.

Reply:
Oops :see_no_evil:. Fixed in https://github.com/safe-global/safe-modules/pull/412/commits/15a68d1a4fee39d15837d1ff5ba2e8703e0de847